<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameSEO</class>
 <widget class="QWidget" name="FrameSEO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBoxSEOTagsVars">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Переменные конструктора автоматического формировавния мета тегов</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Заголовок (title)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditSEOHeader"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Ключевые слова (keywords)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditSEOKeywords"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Оглавление страницы (H1)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditPageTitle"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Описание (description)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="plainTextEditSEODescription"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
